buff.lic

```diff
@@ -21,6 +21,10 @@ class Waggle
 
     waggle_spells = @settings.waggle_sets
 
+    if args.spells['prehunt_buffs']
+      return unless waggle_spells[args.spells]
+    end
+
```

Comment:
Yes, this didn't seem very good to me either. This makes sense though, checking for the spells in waggle_spells. That's better and makes way more sense.